src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AncientAltarListener.java

```diff
@@ -357,4 +358,11 @@ public void onBlockPlace(BlockPlaceEvent e) {
         return Optional.empty();
     }
 
+    @EventHandler(priority = EventPriority.LOW, ignoreCancelled = true)
+    public void onItemDespawn(ItemDespawnEvent e) {
+        if (AncientPedestal.testItem(e.getEntity())) {
+            e.setCancelled(true);
+        }
+    }
+
```

Comment:
```suggestion
```

Reply:
I honestly left it because I wasn't sure which way fit the code style. A lot of classes have an empty line before the closing bracket and a lot also don't, this one did before hand so I left it